docs/development/howto.rst

```diff
@@ -74,6 +71,22 @@ Skip unit tests for some Astropy versions
    def test_something():
       ...
 
+Making a pull request with new or modified datasets
+---------------------------------------------------
+
+Datasets used in tests and tutorials are hosted in the
+`gammapy-data Github repository <https://github.com/gammapy/gammapy-data>`__. It is recommended that developers
+have `$GAMMAPY_DATA` environment variable pointing to the local folder where they have fetched the `gammapy-data` Github
+repository, so they can push and pull eventual modification of its content. In the cases they want to make a pull
+request with new or modified datasets, these are the steps to follow:
+
+#. Update the content of the remote specific dataset repository (i.e. `gammapy-extra`, `gammapy-fermi-lat-data`,
+   `gamma-cat`).
+#. Update the content of your local `gammapy-data` repository and JSON index file by runinng
+   `python make_datasets_index.py` in the `dev/datasets/` folder.
+#. Push eventual changes in `gammapy-data` to remote repository and re-tag the `master` tag.
```

Comment:
Why is the master tag needed here? Can't we just download from the master branch?

Reply:
I don't know how to download the bundle tar.gz file without making a *tag* in the repository.

Reply:
I think we can just use the url `https://github.com/gammapy/gammapy-data/tarball/master`...

Reply:
Yes!!
I've pushed a commit with the modifs :)

Reply:
Great thanks!